### ISSUE TITLE
Fotema does not import images or videos from linked folders.

### ISSUE THREAD

**Issue by NelvinBHV** [blissd/fotema#114]
I have included a folder as a symbolic link in the Pictures folder. Unfortunately, Fotema completely ignores it. The symbolic link points to a Samba share.

**Comment by blissd** [comment]
Hi NelvinBHV, thank you for your bug report. However, I don't think this is a bug, but by design. Fotema's Flatpak permissions only allow reading from your pictures directory, so it won't be able to read pictures from another location pointed at by the symbolic link. 

You could try using the Flatseal app to add another directory to those that Fotema can read. I'm not sure if Fotema will follow the symbolic link when walking the directory tree, but you can give it a try.

**Comment by NelvinBHV** [comment]
Hi blissd,
this patch modifies the core/src/photo/scanner.rs and core/src/video/scanner.rs files to enable following symbolic links by adding the .follow_links(true) method to the WalkDir instance.
`diff --git a/core/src/photo/scanner.rs b/core/src/photo/scanner.rs
index e69de29..a34e843 100644
--- a/core/src/photo/scanner.rs
+++ b/core/src/photo/scanner.rs
@@ -1,1 +1,1 @@
 let walker = WalkDir::new(&self.scan_base)
+    .follow_links(true)
     .into_iter()
     .filter_entry(|e| !is_hidden(e));

diff --git a/core/src/video/scanner.rs b/core/src/video/scanner.rs
index e69de29..a34e843 100644
--- a/core/src/video/scanner.rs
+++ b/core/src/video/scanner.rs
@@ -1,1 +1,1 @@
 let walker = WalkDir::new(&self.scan_base)
+    .follow_links(true)
     .into_iter()
     .filter_entry(|e| !is_hidden(e));
`

**Comment by blissd** [comment]
Be aware that if your have pictures on a network share and it isn't mounted when Fotema starts, then Fotema will see consider those pictures as deleted and will perform some database maintenance to remove the records for the missing files and will also delete any thumbnails that were generated for the missing images. If the network share re-appears and you start Fotema again, then it will see all the pictures as new files and generate the thumbnails again.

**Comment by NelvinBHV** [comment]
Of course, that's correct, but it behaves like any folder that is removed and then re-added. Alternatively, it would also help if Fotema allowed the addition of other folders containing images.

**Comment by CocolinoFan** [comment]
Someone might find this useful. 
If you are using the Flatpak version of Fotema, it will look for pictures in `xdg-pictures:ro`. You can use the program Flatseal to add directories that Fotema can access. I find the behaviour of that to be inconsistent sometimes pulling pictures from one directory up (not sure tho, needs more testing).
I found it useful to modify what `xdg-pictures:ro` is. From the [documentation](https://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access)  it provides access to `$XDG_PICTURES_DIR` or `$HOME/Pictures`. You can change [$XDG_PICTURES_DIR](https://wiki.archlinux.org/title/XDG_user_directories) in `$HOME/.config/user-dirs.dirs`
